sql/src/test/java/io/crate/integrationtests/SysSegmentsTableInfoTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.integrationtests;
+
+import org.elasticsearch.Version;
+import org.junit.Test;
+
+import java.util.Map;
+
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.core.Is.is;
+
+
+public class SysSegmentsTableInfoTest extends SQLTransportIntegrationTest {
+
+    @Test
+    public void test_retrieve_segment_information() {
+        execute("create table t1 (id INTEGER, name STRING) clustered into 1 shards with (number_of_replicas = 1)");
+        ensureYellow();
+        execute("insert into t1 values (1, 'test')");
+        refresh();
+        execute("select table_name, shard_id, segment_name, generation, num_docs, deleted_docs, size, memory, " +
+                "committed, search, compound, version, node, attributes from sys.segments");
+
+        assertThat(response.rowCount(), greaterThan(0L));
+        for (Object[] r : response.rows()) {
+            validateResponse(r);
+        }
+    }
+
+    private void validateResponse(Object[] result) {
+        assertThat(result[0], is("t1"));
+        assertThat(result[1], is(0));
+        assertThat(result[2], is("_0"));
+        assertThat(result[3], is(0L));
+        assertThat(result[4], is(1));
+        assertThat(result[5], is(0));
+        assertThat((Long) result[6], greaterThan(3000L));
```

Comment:
i am just curios about `3000L` how is that calculated?

Reply:
1 Segment with 1 small document including all the meta information turns out to be around 3 kb, depending on the fs/storage.